### PR TITLE
[No Merge] Merge SSE and SSE2 classes

### PR DESCRIPTION
**Pull request by fiigii**
For discussing https://github.com/dotnet/corefx/issues/29247

cc @CarolEidt @tannergooding @eerhardt @AndyAyersMS @jkotas

### CODE REVIEW

File: src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs [hunk 1]
```diff
@@ -719,141 +719,18 @@ public static class Popcnt
...
-        public static Vector128<byte> Add(Vector128<byte> left, Vector128<byte> right) { throw null; }
-        public static Vector128<sbyte> Add(Vector128<sbyte> left, Vector128<sbyte> right) { throw null; }
-        public static Vector128<short> Add(Vector128<short> left, Vector128<short> right) { throw null; }
-        public static Vector128<ushort> Add(Vector128<ushort> left, Vector128<ushort> right) { throw null; }
-        public static Vector128<int> Add(Vector128<int> left, Vector128<int> right) { throw null; }
-        public static Vector128<uint> Add(Vector128<uint> left, Vector128<uint> right) { throw null; }
-        public static Vector128<long> Add(Vector128<long> left, Vector128<long> right) { throw null; }
-        public static Vector128<ulong> Add(Vector128<ulong> left, Vector128<ulong> right) { throw null; }
-        public static Vector128<double> Add(Vector128<double> left, Vector128<double> right) { throw null; }
+        public static Vector128<T> Add<T>(Vector128<T> left, Vector128<T> right) where T : struct { throw null; }
```
Comment (tannergooding) [inline src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs]: nit: If these were merged (and I still **don't** think it is a good idea), most of these would/could become generic.

**Comment by 4creators** [comment]
@fiigii I agree with @tannergooding that it is not good idea to merge them.

**Comment by stephentoub** [comment]
@fiigii, @CarolEidt, this can be closed now, right?

**Comment by fiigii** [comment]
@stephentoub The discussion https://github.com/dotnet/corefx/issues/29247 is still in progress and this PR is a demo for that thread. Once https://github.com/dotnet/corefx/issues/29247 gets a consensus, I will close this PR.

**Comment by stephentoub** [comment]
Thanks.  But a PR isn't needed for that, though; the discussion happens on the issue, and you can point people to your branch to see what it looks like (https://github.com/fiigii/corefx/commits/mergesse/). Having two places for the discussion to happen seems worse to me, plus it clutters up the PR list with a PR that won't actually be merged.

**Comment by fiigii** [comment]
@stephentoub Thanks for the explanation, closing.

**Comment by stephentoub** [comment]
Thanks, @fiigii.

**Pull request closed without merging**